test/module/shared_model/validators/transaction_validator_test.cpp

```diff
@@ -101,13 +101,25 @@ TEST_F(TransactionValidatorTest, StatelessValidTest) {
                    },
                    [] {});
 
-  shared_model::validation::DefaultTransactionValidator transaction_validator;
   auto result = proto::Transaction(iroha::protocol::Transaction(tx));
   auto answer = transaction_validator.validate(result);
 
   ASSERT_FALSE(answer.hasErrors()) << answer.reason();
 }
 
+/**
+ * @given Protobuf transaction object with unset command
+ * @when validate is called
+ * @then there is a error returned
+ */
+TEST_F(TransactionValidatorTest, UnsetCommand) {
+  iroha::protocol::Transaction tx = generateEmptyTransaction();
+  tx.mutable_payload()->set_creator_account_id(account_id);
+  tx.mutable_payload()->set_created_time(created_time);
+  auto answer = transaction_validator.validate(proto::Transaction(tx));
+  ASSERT_TRUE(answer.hasErrors());
```

Comment:
Answer here is `Transaction: [[Transaction should contain at least one command ]]`, which is not the described case.
There should be an additional call `tx.mutable_payload()->add_commands();`